exercise-core/src/test/java/com/examples/attsw/exercise/core/controller/EmployeeControllerTest.java

```diff
@@ -0,0 +1,63 @@
+package com.examples.attsw.exercise.core.controller;
+
+import static org.junit.Assert.*;
+
+import java.util.LinkedList;
+import java.util.List;
+
+import org.junit.Before;
+import org.junit.Test;
+import org.mockito.Mockito;
+
+import com.examples.attsw.exercise.core.model.Employee;
+import com.examples.attsw.exercise.core.service.IEmployeeService;
+
+public class EmployeeControllerTest {
+
+	// SUT
+	public EmployeeController employeeController;
+	private List<Employee> listEmployees;
+	private IEmployeeService iEmployeeService;
+
+	@Before
+	public void setUp() throws Exception {
+		iEmployeeService = Mockito.mock(IEmployeeService.class);
+		employeeController = new EmployeeController(iEmployeeService);
+		listEmployees = new LinkedList<Employee>();
+		Mockito.when(iEmployeeService.getEmployees()).thenReturn(listEmployees);
+	}
+
+	@Test
+	public void testGetAllEmployeeWhenThereAreNoEmployee() {
+		assertGetAllEmployee("");
+	}
+
+	@Test
+	public void testGetAllEmployeeWhenThereIsOneEmployee() {
+		Employee employee = newEmployee("nameTest", "idTest");
+		listEmployees.add(employee);
+		assertGetAllEmployee(employee.getName());
+	}
```

Comment:
when dealing with lists, first test the case when the list is empty (OK), then with one element (OK) and then with two elements which is not tested at all, in fact, the implementation for several elements does not respect the specification of the issue (see the above comment)